sale_service_project/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Service Project
+====================
+
```

Comment:
This module helps you to manage task created from sales order, let you choose if you wants to invoice from sale order or from task, and helps you to control with sale order must be invoiced or not depending if the task have been done or not.